### ISSUE TITLE
Python 3.7.4 Dockerfile is missing from this repository

### ISSUE THREAD

**Issue by michaelosthege** [jonemo/pythonnet-docker#1]
Hi @jonemo , thank you for building these images!!

On Docker Hub there is a Tag for Python 3.7.4 (https://hub.docker.com/layers/jonemo/pythonnet/python3.7.4-mono5.20-pythonnet2.4.0/images/sha256-3fd3fb542e8c0144201390283620fb4b4be9ab9103b9390d1a7ace3edfcf988e?context=explore)

But the Dockerfile does not seem to be present in this repository?

**Comment by jonemo** [comment]
Hi @michaelosthege, you got me there! 

Last time I spend time on this project I was in a rush and only uploaded images to Dockerhub but then forgot to update this repo. I just went back, reviewed and cleaned up the code, and pushed the latest version from 10 months ago. Thank you for the reminder!

The file you are looking for is now here: https://github.com/jonemo/pythonnet-docker/blob/master/archived-dockerfiles/python3.7.4-mono5.20-pythonnet2.4.0

Of course, now that I've got this code open, I might as well build a few new images...
